mapservutil.c

```diff
@@ -1530,7 +1530,7 @@ int msCGIDispatchImageRequest(mapservObj *mapserv)
     case TILE:
       msTileSetExtent(mapserv);
 
-      if(!strcmp(MS_IMAGE_MIME_TYPE(mapserv->map->outputformat), "application/x-protobuf")) {
+      if(!strcmp(MS_IMAGE_MIME_TYPE(mapserv->map->outputformat), "application/vnd.mapbox-vector-tile")) {
```

Comment:
you could accept both outputformats here for backward compatibility

Reply:
Both are accepted now, thanks